lib/internal/crypto/pbkdf2.js

```diff
@@ -22,12 +22,12 @@ function pbkdf2(password, salt, iterations, keylen, digest, callback) {
     digest = undefined;
   }
 
-  ({ password, salt, iterations, keylen, digest } =
-      check(password, salt, iterations, keylen, digest, callback));
-
   if (typeof callback !== 'function')
     throw new ERR_INVALID_CALLBACK();
 
+  ({ password, salt, iterations, keylen, digest } =
+    check(password, salt, iterations, keylen, digest));
+
```

Comment:
Why did you move this here? Why not just keep it above the `callback` check?

Reply:
My mistake. I'll fix that soon.